src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -135,4 +173,36 @@ private static void remove(Block b) {
             hologram.remove();
         }
     }
+
+    private void createSelector(int menuSlot, ItemStack button, char coordinate, char direction, String parameter, float smallChange, float largeChange, Block projector, ChestMenu menu) {
+
+        menu.addItem(menuSlot, new CustomItem(button,
+            "&7" + coordinate + " Offset (" + direction + ") &e" + DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), parameter))),
+            "", "&eLeft Click: &7+0.1", "&eRight Click: &7+0.5"));
+        menu.addMenuClickHandler(menuSlot, (pl, slot, item, action) -> {
+            double storedOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), parameter));
+            double offset;
+
+            offset = action.isRightClicked() ? largeChange : smallChange;
+
+            if ((coordinate == 'X' || coordinate == 'Z')
+                && offset + storedOffset > maxDistance.getValue()) {
+                return false;
+            }
+            ArmorStand hologram = getArmorStand(projector, true);
+
+            if (coordinate == 'X') {
+                hologram.teleport(hologram.getLocation().add(offset, 0, 0));
+            } else if (coordinate == 'Y') {
+                hologram.teleport(hologram.getLocation().add(0, offset, 0));
+            } else if (coordinate == 'Z') {
+                hologram.teleport(hologram.getLocation().add(0, 0, offset));
+            }
+
+            BlockStorage.addBlockInfo(projector, parameter,
+                String.valueOf(storedOffset + offset));
```

Comment:
Check which one of the offsets are bigger than like 0.05 and set its value.

Reply:
This just shifts the if/else statements down.. and feels jankier?

Reply:
This was a continuation of the change requested above.

Reply:
I know

Reply:
Now that I think about it, hmm yes that might be right. A char check is checking 2 bytes, whereas double check is mucu more complicated. Okay then keep it likewise. I'll review this with a fresh mind tomorrow.